nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -74,6 +74,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [appvm](https://github.com/jollheef/appvm), Nix based app VMs. Available as [virtualisation.appvm](options.html#opt-virtualisation.appvm.enable).
 
+- [blesh](https://github.com/akinomyoga/ble.sh), a line editor written in pure bash. Avilable as [programs.bash.blesh](#opt-programs.bash.blesh.enable).
```

Comment:
```suggestion
- [blesh](https://github.com/akinomyoga/ble.sh), a line editor written in pure bash. Avilable as [programs.bash.blesh](#opt-programs.bash.blesh.enable).

```

Reply:
What change do you suggest? i see no difference. I can only guess you meant to fix Avilable -> Available

Reply:
And you introduced a blank line. didn't spot that at first sight.

Reply:
GitHub can't display that properly...